src/docs/product/accounts/quotas/index.mdx

```diff
@@ -206,7 +253,41 @@ This feature is available only if your organization is on a Business plan.
 
 If you have enabled the storage of crash reports, you may set limits for the maximum number of crash reports that will be stored per issue. To set up these limits, use the slider in the "Store Native Crash Reports" option in your organization's "Security & Privacy" settings.
 
-## Inbound Filters {#inbound-data-filters}
+## 5. Delete, & Discard Issues
+
+<Note>
+
+This feature is available only on a Business plan or higher.
+
+</Note>
+
+If there is an irrelevant, reoccurring issue that you are unable or unwilling to resolve, you can delete and discard it from the **Issue Details** page by clicking "Delete and discard future events". This will remove the issue and event data from Sentry and filter out future matching events.
+
+![Delete and Discard](manage-event-stream-06.png)
+
+You can find a list of these issues in the "Discarded Issues" tab in **[Project] > Settings > Inbound Filters**. From here, you can un-discarded any of these issues to receive future events.
+
+![List of Discarded Issues](manage-event-stream-07.png)
+
+<Note>
+
+Once you've identified a set of discarded issues, it might make sense to go back to your SDK configuration and add the related errors into your `before-send` client-side filtering.
+
+</Note>
+
+### Event Repetition
+
+Event repetition only applies to error events, not transactions or attachments. The following rules apply for event repetition and your quota:
+
+- If you have chosen to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota.
+- If you previously resolved an issue, a new error event counts toward your quota because it may represent a regression in your code.
+- If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#inbound-data-filters).
```

Comment:
```suggestion
- If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#6-inbound-filters).
```